app/controllers/imports/issues_controller.rb

```diff
@@ -53,7 +53,7 @@ def set_import_issues
   def set_record
     @record =
       if params[:type] == "vaccination-record"
-        @vaccination_records.find(params[:id])
+        @vaccination_records.unscoped.find(params[:id])
```

Comment:
Rather than `unscoped`, I think you'll want to use [`with_discarded`](https://github.com/jhawthorn/discard?tab=readme-ov-file#default-scope), otherwise it'll affect more than just the default scope of `kept`.